fineract-provider/build.gradle

```diff
@@ -450,6 +472,7 @@ modernizer {
 
 compileJava{
     dependsOn rat
+    dependsOn spotlessCheck
 }
```

Comment:
@vorburger  FYI, I added this to the commit(on Friday) for making spotless run before compile java as you suggested. The other change in the readme(explaning about .XML). Everything else is the same.

Reply:
Great, thanks. FYI I'm going to be merging this shortly (started working on it right now).